### DISCUSSION TITLE
Chrome Extension?

### DISCUSSION THREAD

**Discussion by tpavlu** [jotaijs/jotai-devtools#111] [Ideas]
I was curious if there were any plans to make a chrome extension based implementation of these tools?
I find them helpful but sometimes its hard to manage screen real estate with the actual application.

I was considering just creating an extension that got a dynamic reference the store like the redux extension and then just re-used this react code but I didn't want to re-invent the wheel if this group was looking to support that eventually.

**Comment by arjunvegda** [comment]
Hey @tpavlu 

We have an ongoing (somewhat stale now 😓) discussion on this. Please see: https://github.com/jotaijs/jotai-devtools/discussions/59

**Reply by tpavlu** [reply]
Thanks for the redirect I will contribute there!
